cypress/page/products-list.page.ts

```diff
@@ -0,0 +1,19 @@
+class ProductsListPage {
+  private addToCart: string;
+  private containerCart: string;
+  private product: string;
+
+  constructor() {
+    this.containerCart = ".available-now";
+    this.addToCart = ".button.ajax_add_to_cart_button.btn.btn-default";
+    this.product = ".button-container span .icon-chevron-right";
+  }
+
+  public goToAddToCart(): void {
+    cy.get(this.containerCart).click();
+    cy.get(this.addToCart).click();
+    cy.get(this.product).click();
+  }
```

Comment:
I would split this method in two:

1. One method to click in the add to cart button.
2. The second method to click in the proceed to checkout button.